core/src/main/java/org/mule/runtime/core/api/store/DeserializationPostInitialisable.java

```diff
@@ -4,7 +4,7 @@
  * license, a copy of which has been included with this distribution in the
  * LICENSE.txt file.
  */
-package org.mule.runtime.core.util.store;
+package org.mule.runtime.core.api.store;
```

Comment:
are we okay with this being api? isn't this going to be removed/replaced eventually?